app/views/layout.scala.html

```diff
@@ -22,20 +22,21 @@
 @import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcStandardHeader
 @import uk.gov.hmrc.govukfrontend.views.html.components.GovukLayout
 @import config.ApplicationConfig
+@import views.html.helper.CSPNonce
 
 @this(
         appConfig: ApplicationConfig,
-        govukLayout: GovukLayout,
+        hmrcLayout: HmrcLayout,
         hmrcHead: HmrcHead,
         hmrcScripts: HmrcScripts,
         govukHeader: HmrcStandardHeader,
         problemLink: HmrcReportTechnicalIssue,
         hmrcTimeoutDialogHelper: HmrcTimeoutDialogHelper,
-        languageSelection: hmrcComponents.hmrcLanguageSelect,
         accessibilityStatementConfig: AccessibilityStatementConfig,
         hmrcTrackingConsentSnippet: HmrcTrackingConsentSnippet
 )
 
+
 @(pageTitle: Option[String] = None, scripts: Option[Html])(contentBlock: Html)(implicit request: Request[_], messages: Messages)
 
 @headBlock = {
```

Comment:
I think we can now remove the headBlock since all that is needed for the tracking consent is the nonce = CSPNonce.get in the @hmrcLayout